docs/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Description
+<!--- Please describe shortly what this pull request ist doing. If there is an related issue, please mention it here. -->
+
+## Checklist
+<!--- Please make sure to go over all points on the checklist and mark them as checked. -->
+- [ ] I made sure that the markdown files are formatted properly.
+- [ ] I made sure that all hyperlinks are working.
+
+If the pull request adds new content
```

Comment:
```suggestion
If the pull request adds new content: <!-- Please remove the following lines otherwise -->
```

Reply:
I tried to make it even more clear. :smile: 
I also made you Co-Author so you get credit.